code/index.html

```diff
@@ -1,30 +1,129 @@
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Munchies - Book a cooking class</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <!--Hero section-->
+  <header class="header">
+    <div class="navigation">
+      <nav class="navbar-right">
+        <a href="#">about us</a>
+        <a href="#faq-section">faq</a>
+        <a href="#submit-form">Sign me up</a>
+      </nav>
+    </div>
+  </header>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+  <article>
+    <div class="head-container">
+      <div class="head-content">
+        <div class="head-overlay">
+          <h1>Join our authentic cooking class</h1>
+          <a href="#submit-form" class="welcome-button">BOOK A TIME HERE</a>
+        </div>
       </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
-</html>
+      <div class="head-video">
+        <video autoplay loop muted playsinline>
+          <source src="videos/video.mp4" type="video/mp4">
+          Your browser does not support the video tag.
+        </video>
+      </div>
+    </div>
+
+    <!--FORM SECTION-->
+    <div class="main-content">
+      <section class="form-wrapper" id="submit-form">
+        <h2>Sign-up here</h2>
+        <form action="https://httpbin.org/anything" method="POST">
+          <fieldset>
+            <legend>Book a time</legend>
+            <div class="labels">
+              <label for="firstname">First Name:</label>
+              <input type="text" id="firstname" name="user_firstname" class="form-control"
+                placeholder="Enter your first name" required />
+            </div>
+            <div class="labels">
+              <label for="lastname">Last name:</label>
+              <input type="text" id="lastname" name="user_lastname" class="form-control"
+                placeholder="Enter your last name" required>
+            </div>
+            <div class="labels">
+              <label for="mail">E-mail address:</label>
+              <input type="email" id="mail" name="user_mail" class="form-control" placeholder="Enter your email"
+                required>
+            </div><br>
+            <div>
+              <p>Pick a time slot:</p>
```

Comment:
Maybe an [input with date type ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date)would be suited here?